.travis/update-version.sh

```diff
@@ -13,7 +13,7 @@ update_version() {
     mvn --settings .travis/settings.xml org.codehaus.mojo:versions-maven-plugin:2.7:set -DnewVersion="${TRAVIS_TAG}-SNAPSHOT"
```

Comment:
do we also need to specify the default build profile here?

Reply:
No that runs on all of them but I can just to be safe